universal-login-sdk/lib/sdk.ts

```diff
@@ -200,7 +203,19 @@ class UniversalLoginSDK {
   async connect(walletContractAddress: string) {
     const {publicKey, privateKey} = createKeyPair();
     await this.relayerApi.connect(walletContractAddress, publicKey.toLowerCase());
-    return privateKey;
+    const securityCode = this.securityCodeService.generateCode(publicKey);
+    return {
+      privateKey,
+      securityCode
+    };
+  }
+
+  verifySecurityCode(secuirtyCode: number[], publicKey: string) {
+    return this.securityCodeService.isValidCode(secuirtyCode, publicKey);
+  }
+
+  generateCodeWithFakes(publicKey: string) {
+    return this.securityCodeService.generateCodeWithFakes(publicKey);
   }
 
```

Comment:
About subscribeAuthorisations below

If we make `signature` in `GetAuthorisationRequest` optional we can shorten the function:
```ts
subscribeAuthorisations(walletContractAddress: string, privateKey: string, callback: Function) {    
  return this.authorisationsObserver.subscribe(
    signGetAuthorisationRequest({walletContractAddress}, privateKey), 
    callback);
} 
``` 
I would also propose augmenting notifications wtih code in SDK rather than in AuthorisationsObserver, ie:
```ts
subscribeAuthorisations(walletContractAddress: string, privateKey: string, callback: Function) {    
  return this.authorisationsObserver.subscribe(
    signGetAuthorisationRequest({walletContractAddress}, privateKey), 
   (notifications) => addCodesToNotifications(notifications) );
}
```